doc/packer.txt

```diff
@@ -280,7 +280,7 @@ installs/updates the given plugin.
 Providing the `installer`/`updater` keys overrides plugin type detection, but
 you still need to provide a location string for the name of the plugin.
 
-POST-UPDATE HOOKS                              *packer-plugin-hooks* 
+POST-UPDATE HOOKS                              *packer-plugin-hooks*
```

Comment:
Nitpick: there seem to be a number of "empty" changes to headings in the docs. It's possible that these are whitespace changes (I'm reviewing on my phone, and not sure that the GitHub Android app renders whitespace diffs well), but if they're not deleting dangling whitespace or similar, I'd prefer they be removed to keep the diff clean.

Reply:
I can confirm these are whitespace diffs.My Neovim has been configured to strip trailing whitespace on save; it seems parts of this file got caught in the crossfire. I didn't mean to update this part of the file— would you like me to revert the changes here?

Reply:
No, that's OK. Stripping whitespace is fine. Thanks!